build.sh

```diff
@@ -9,4 +9,4 @@ cd "$base_dir"
 mvn --batch-mode --file server/ clean install
 
 cd "$base_dir/go-client"
-go test -count 1 -p 1 -v -coverprofile=coverage.out ./...
+go test -timeout 20m -count 1 -p 1 -v -coverprofile=coverage.out ./...
```

Comment:
Timeout is necessary in case Docker containers need to be downloaded

Reply:
:hourglass: